external/tika/pom.xml

```diff
@@ -60,6 +60,11 @@ under the License.
 					<groupId>org.slf4j</groupId>
 					<artifactId>slf4j-log4j12</artifactId>
 				</exclusion>
+				<!-- This comes in through the Tika Image Parser module and has an incompatible BSD license -->
```

Comment:
Apache Tika cleared the use of `jai-imageio-core` here: https://issues.apache.org/jira/browse/LEGAL-304.

I'm ok excluding it to make licensing cleaner for stormcrawler, and I don't think there will be large consequences for stormcrawler.

Reply:
Great to have the link!